### ISSUE TITLE
Smaller or Customisable sidebar content size

### ISSUE THREAD

**Issue by zackpollard** [telegramdesktop/tdesktop#1250]
It would be nice to be able to have smaller chat displays in the sidebar so that more chats can be seen at once, if this was a toggle between for example a one line just for the chat title and the current setup, that would be awesome as I currently have a lot of chats going at once and so it gets very hard to follow as they move off of the sidebar view.

**Comment by stek29** [comment]
@auchri This issue is about vertical shrinking of chats in list, not horizontal.
